gradle/github-release.gradle

```diff
@@ -88,11 +88,31 @@ task updateNewVersionChangelog() << {
     }
 }
 
-afterReleaseBuild.dependsOn(":core-android:uploadArchives", ":rides-android:uploadArchives")
+task configureGithub() << {
```

Comment:
nit - I think the `<<` operator is deprecated in gradle

Reply:
Will investigate and address at a future date.